File: references/RndIndoorLaboratoryMaterial.cs
using System;

namespace RnDLaboratoryStock.Core.Models;

public partial class RndIndoorLaboratoryMaterial
{
    public int Id { get; set; }

    public string? Epc { get; set; }

    public string Brand { get; set; } = null!;

    public string BrandCode { get; set; } = null!;

    public double Amount { get; set; }

    public string Unit { get; set; } = null!;

    public string ChemicalName { get; set; } = null!;

    public string CompanyName { get; set; } = null!;

    public int ItemNumber { get; set; }

    public int Remain { get; set; }

    public int? Limit { get; set; }

    public DateTime InsertedAt { get; set; }

    public DateTime? Exp { get; set; }
}

File: app/src/main/java/com/example/rndlaboratorystock/Models/RndIndoorLaboratoryMaterial.java
package com.example.rndlaboratorystock.Models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RndIndoorLaboratoryMaterial {
    @SerializedName("responseCode")
    public int ResponseCode;
    @SerializedName("data")
    public List<Data> Data = null;

    public class Data {
        @SerializedName("id")
        public int Id;
        @SerializedName("epc")
        public String Epc;
        @SerializedName("brand")
        public String Brand;
        @SerializedName("brandCode")
        public String BrandCode;
        @SerializedName("amount")
        public double Amount;
        @SerializedName("unit")
        public String Unit;
        @SerializedName("chemicalName")
        public String ChemicalName;
        @SerializedName("companyName")
        public String CompanyName;
        @SerializedName("itemNumber")
        public int ItemNumber;
        @SerializedName("remain")
        public int Remain;
        @SerializedName("limit")
        public int Limit;
        @SerializedName("insertedAt")
        public String InsertedAt;
        @SerializedName("exp")
        public String Exp;
    }
}
